homework3436/HomeWorkVinny.py

```diff
@@ -0,0 +1,29 @@
+# Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм.
+# Поскольку разобраться в его кричалках не настолько просто,
+# насколько легко он их придумывает, Вам стоит написать программу.
+# Винни-Пух считает, что ритм есть, если число слогов
+# (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+# Фраза может состоять из одного слова, если во фразе несколько слов,
+# то они разделяются дефисами. Фразы отделяются друг от друга пробелами.
+# Стихотворение  Винни-Пух вбивает в программу с клавиатуры.
+# В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке.
+
+
+def check_rhythm(poem):
+    phrases = poem.split(" ")  # Разбиваем стихотворение на отдельные фразы по пробелам
+    syllables = set()  # Создаем множество для хранения уникальных значений слогов
+    
+    for phrase in phrases:
+        words = phrase.split("-")  # Разбиваем каждую фразу на отдельные слова по дефисам
+        # Считаем количество гласных в каждом слове и суммируем их
+        phrase_syllables = sum(word.count(vowel) for word in words for vowel in "аеёиоуыэюяАЕЁИОУЫЭЮЯ")
```

Comment:
Разбивать по дефисам не нужно.
```suggestion
        # Считаем количество гласных в каждом слове и суммируем их
        phrase_syllables = sum(char in "аеёиоуыэюяАЕЁИОУЫЭЮЯ" for char in phrase)
```
Подумайте, что я сделал.

Reply:
И, кстати, можно входную строку перевести к одному регистру, тогда не надо будет использовать два регистра в строке с гласными